config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  root 'users#index'
+
+  resources :users, only: %i[index show]
```

Comment:
[OPTIONAL]

- [ ] To create well looking routes and making sure that a post will be created on behalf of a user, we put a post resources inside that of a user.
```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```